iosApp/iosApp/ContentView.swift

```diff
@@ -170,13 +171,26 @@ struct ContentView: View {
     }
 
     @ViewBuilder var mapSection: some View {
-        HomeMapView(
-            contentVM: contentVM,
-            mapVM: mapVM,
-            nearbyVM: nearbyVM,
-            viewportProvider: viewportProvider,
-            sheetHeight: $sheetHeight
-        )
+        // If we don't have an access token, don't attempt to show the map
+        switch onEnum(of: contentVM.configResponse) {
+        case .ok:
```

Comment:
I think we *would* want to show the map if the user doesn't have the access token, because there could be good reason they don't have an access token: they might be in a subway tunnel and have no access. With the current implementation where the map is always shown, a user can still see the stops, route shapes, and cached map tiles while offline.